pkgs/applications/misc/html5validator/default.nix

```
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchFromGitHub, nose, openjdk, stdenv }:
+buildPythonPackage rec {
```

it's not being exported in python-modules, so need to export library
```suggestion
buildPythonApplication rec {
```